pkg/machines/scripts/install_machine.sh

```diff
@@ -73,6 +73,7 @@ virt-install \
     --wait -1 \
     --noautoconsole \
     --noreboot \
+    --check path_in_use=off \
```

Comment:
This needs explanation. It's not only affecting what this PR is supposed to do.

Reply:
Actually, this parameter is useful only when supplying already existing disk (what this PR does).
It won't affect anything but this PR, and It won't change behavior in any other use cases, since in all the other use cases we create our own disks for every VM creation, so checking if disk is already used by another VM is useless.
But I can leave comment "allow user tor supply disk used by another VM" if you want.

Reply:
What about when importing disks...? That option used to exist previously as well.

Reply:
for importing disk, that will be prevented in create_machine.sh, so I guess there is no need to also check it in install_machine.sh ?